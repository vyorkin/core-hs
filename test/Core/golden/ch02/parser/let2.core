f = \x. (let y = \z. z; g = 1; w = 3 in y g)
