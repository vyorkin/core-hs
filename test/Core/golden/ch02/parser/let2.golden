[(Name "f",[],ELam [Name "x"] (ELet False [Defn (Name "y") (ELam [Name "z"] (EVar (Name "z"))),Defn (Name "g") (ENum 1),Defn (Name "w") (ENum 3)] (EAp (EVar (Name "y")) (EVar (Name "g")))))]